### ISSUE TITLE
WFC does not display correct details on Server 2008.

### ISSUE THREAD

**Issue by BloodStoneBBS** [wwivbbs/wwiv#183]
Windows Server 2008 x86, WWIV 5.00 Build 1552 (and previous going back to 15xx)
Example: BBS up for several weeks, networked to WWIVnet, the [Messages Today], [Local Messages Today], [Email Today], [Feedback Today] and [Mins Used Today] fields reflect total since install. 

![capture1](https://cloud.githubusercontent.com/assets/13243281/9798466/e3868b5c-57b5-11e5-9b1e-d74b87698568.JPG)

BTW, this instance of the BBS had not been running since yesterday @ 3:00 AM.

**Comment by BloodStoneBBS** [comment]
Now on build 1567 -

A few days later shows increases in stats but are cumulative rather than per date. 
Disregard OS, This is actually on WIndows Server 2008 x86

![capture2](https://cloud.githubusercontent.com/assets/13243281/9855202/a09fe7b6-5ac1-11e5-80fe-614d25580f3a.JPG)

**Comment by BloodStoneBBS** [comment]
Does this ionformation come from STATUS.DAT? Fix log from 9/10 shows date errors corrected.
Just ran FIX again, showing date errors corrected again.
![capture](https://cloud.githubusercontent.com/assets/13243281/9855661/997a1e72-5ac4-11e5-8233-16b25de2800b.JPG)

**Comment by wwiv** [comment]
Looks like the begin day event didn't run

**Comment by BloodStoneBBS** [comment]
Agreed.. Still /E ?

**Comment by BloodStoneBBS** [comment]
Manually ran bbs.exe /I6 /E
Received WARNING: Tried to run beginday event again
/N and /I the same?

**Comment by BloodStoneBBS** [comment]
Are *.STR files still required? They are not in my BBS/GFILES folder on Wondows 2008 as they are in my Win7 box

**Comment by askpatrickw** [comment]
They must be there from a previous install. They are not on my machine.

**Comment by wwiv** [comment]
correct ,wwiv doesn't use the str files anymore.

**Comment by wwiv** [comment]
/n and /i are the same.
bbs /e warned you since the last run date matches the current date (fix fixed that). 
if you see this tomorrow let me know

**Comment by BloodStoneBBS** [comment]
Will do. It's been this way since I installed on server 2008. Not critical but it does make me wonder.

**Comment by BloodStoneBBS** [comment]
No idea what I did but the WFC stats are correct this morning on 2008 server.
Could moving the network from \bbs\networks\wwivnet to \bbs\data\wwivnet have done anything?
(testing to see if it was easy to move the netdata before cleaning up my live BSBBS)
That's about all I did on that install over the last 48 hours.
I think it's probably fine to close this now afaik I am the only one who experienced this and was probably something I did/didn't do.
